pkg/models/resources/v1alpha3/secret/secrets.go

```diff
@@ -74,5 +76,10 @@ func (s *secretSearcher) filter(object runtime.Object, filter query.Filter) bool
 		return false
 	}
 
+	if filter.Field == query.ParameterFieldSelector &&
+		strings.Contains(string(filter.Value), string(secret.Type)) {
```

Comment:
Why not use `jsonpath.JsonPathLookup` instead? It can be a generic query filter, just go ahead. As we know, the field selector is still not working well with custom resources, https://github.com/kubernetes/kubernetes/issues/51046.

Reply:
Is this what you mean? https://github.com/oliveagle/jsonpath

Reply:
> Why not use `jsonpath.JsonPathLookup` instead? It can be a generic query filter, just go ahead. As we know, the field selector is still not working well with custom resources, [kubernetes/kubernetes#51046](https://github.com/kubernetes/kubernetes/issues/51046).

ok, done